Formula/proxysql.rb

```diff
@@ -0,0 +1,37 @@
+class Proxysql < Formula
+  desc "High Performance Advanced Proxy for MySQL"
+  homepage "http://www.proxysql.com/"
+  url "https://github.com/sysown/proxysql/archive/v1.4.10.tar.gz"
+  sha256 "29020cba75b778ef45b06c5469ee5930ce80afbd0290a112d8c2a6652b4c8a2e"
+
+  # Build dependencies listed here: https://github.com/sysown/proxysql/blob/master/INSTALL.md
+  depends_on "automake" => :build
+  depends_on "cmake" => :build
+  depends_on "make" => :build
+  depends_on "mysql@5.6" => :test
+  depends_on "openssl"
+
+  def install
+    system "make"
```

Comment:
I noticed upstream bundles a large chunk of its own deps, including `openssl`. Is it actually using our `openssl` or the bundled static one or both or?

Reply:
> If you'd like me to spend more time on that, I'm happy to--just let me know.

No need. I'd prefer this to always use our OpenSSL, so we're not waiting on upstream to catch up to security updates.

Reply:
👍